templates/openstack/_resource-card.html

```diff
@@ -0,0 +1,20 @@
+{% with 
+  title=item.topic_name,
+  title_link=item.path,
+  description=item.subtitle,
+  type=item.type
+%}
+<div class="col-4 blog-p-card--post">
+  <header class="blog-p-card__header">
+    <h5 class="p-muted-heading u-no-margin--bottom">
+      {{ type }}
+    </h5>
```

Comment:
I don't think these need to be heading elements, and the heading heirarchy jumps from `h3` to `h5` then back to `h4`:
```suggestion
    <p class="p-muted-heading u-no-margin--bottom">
      {{ type }}
    </p>
```